diff --git a/dmenu_path b/dmenu_path
index 3a7cda7..bbdff71 100755
--- a/dmenu_path
+++ b/dmenu_path
@@ -1,13 +1,17 @@
 #!/bin/sh
 
+# UPDATE PATH HERE
+dmenu_dir="$HOME/suckless/dmenu/"
+stest_bin="$dmenu_dir/stest"
+
 cachedir="${XDG_CACHE_HOME:-"$HOME/.cache"}"
 cache="$cachedir/dmenu_run"
 
 [ ! -e "$cachedir" ] && mkdir -p "$cachedir"
 
 IFS=:
-if stest -dqr -n "$cache" $PATH; then
-	stest -flx $PATH | sort -u | tee "$cache"
+if $stest_bin -dqr -n "$cache" $PATH; then
+	$stest_bin -flx $PATH | sort -u | tee "$cache"
 else
 	cat "$cache"
 fi
diff --git a/dmenu_run b/dmenu_run
index 834ede5..917871f 100755
--- a/dmenu_run
+++ b/dmenu_run
@@ -1,2 +1,8 @@
 #!/bin/sh
-dmenu_path | dmenu "$@" | ${SHELL:-"/bin/sh"} &
+
+# UPDATE PATH HERE
+dmenu_dir="$HOME/suckless/dmenu/"
+dmenu_bin="$dmenu_dir/dmenu"
+dmenu_path_bin="$dmenu_dir/dmenu_path"
+
+$dmenu_path_bin | $dmenu_bin "$@" | ${SHELL:-"/bin/sh"} &
